llvm/test/SYCLLowerIR/PropagateAspectsUsage/sycl-fixed-targets.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: opt -passes=sycl-propagate-aspects-usage -sycl-propagate-aspects-usage-fixed-targets=host,cpu,gpu %s -S | FileCheck %s
+
+source_filename = "main.cpp"
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64"
+target triple = "spir64-unknown-unknown"
+
+; CHECK: void @_ZTSZZ4mainENKUlRN4sycl3_V17handlerEE_clES2_EUlvE_(){{.*}}!sycl_fixed_targets ![[#MDNUM:]]{{.*}}
```

Comment:
```suggestion
; CHECK: void @_ZTSZZ4mainENKUlRN4sycl3_V17handlerEE_clES2_EUlvE_(){{.*}}!sycl_fixed_targets ![[#MDNUM:]]
```

No need in trailing `{{.*}}`, it is kind of assumed by default by `FileCheck`.
Also: for the test simplicity/readability the kernel function can be renamed to something like `kernel`